pkgs/development/libraries/theft/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "0.4.3";
+  name = "theft-${version}";
+
+  src = fetchFromGitHub {
+    owner = "silentbicycle";
+    repo = "theft";
+    rev = "v${version}";
+    sha256 = "1ibh8np12lafnrsrvjbbzlyq45zq654939x0y22vdnc6s8dpbhw4";
+  };
+
+  preConfigure = "patchShebangs ./scripts/mk_bits_lut";
+  # doCheck = true;
+  
+  propagatedBuildInputs = [ ];
+
+  buildInputs = [ ];
+
+  installFlags = [ "PREFIX=$(out)" ];
+  postInstall = ''
+      install -m644 vendor/greatest.h $out/include/
+    '';
+  
+  meta = {
```

Comment:
If you change this to ```meta = with stdenv.lib; {``` you can get rid of all the ```stdenv.lib``` stuff below.

Reply:
Understood, but are you indicating this as a helpful hint, or is this a requirement for acceptance of this package?  If it's not required, my preference would be for the current explicit form, but I will make this change if required.

Reply:
It's pretty subjective, so no, not a requirement. I however, find it easier to read.